### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by yazidsef**

### CODE REVIEW

**Review by LaserHuy — APPROVED** [review]
Good jobs bro

File: contact.html [hunk 1]
```diff
@@ -0,0 +1,103 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
```
Comment (Benja-san) [inline contact.html]: name the page title

File: contact.html [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+</head>
+
+<body>
+    <h1 class="center">Contact</h1>
+    <div class="contact">
```
Comment (Benja-san) [inline contact.html]: use Address tag instead of div tag

File: contact.html [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+            <p> 68 av philipe augueste 75011 paris</p>
+
+        </div>
+
+
+    </div>
+    <form action="#" method="post">
+        <fieldset>
+            <legend>Nom et prenom</legend>
+            <input type="text" name="" id="">
```
Comment (Benja-san) [inline contact.html]: if you don't use attributes, please suppress'em

File: contact.html [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+            <textarea name="message" id="" cols="30" rows="10">
+
+            </textarea>
+        </fieldset>
+        <input type="submit" value="Contacter-nous">
+    </form>
+    <h2 class="center">
+        Oxygen Team
+    </h2>
+    <div class="team">
```
Comment (Benja-san) [inline contact.html:53]: use html semantic like section tag

File: contact.html [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+
+        <div class="teamMember">
+            <img src="img\image 1.png" alt="">
+            <h5>Hazar</h5>
```
Comment (Benja-san) [inline contact.html]: where is the h3 ? and the h4?

File: favoris.html [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Favoris</title>
+</head>
+
+<body>
+    <div class="favoris">
+        <h2 class="titles center">
```
Comment (Benja-san) [inline favoris.html]: h1

File: contact.html [hunk 7]
```diff
@@ -0,0 +1,103 @@
...
+                    <img src="img\git.png" alt="">
+                    <img src="img\in.png" alt="">
+                </span>
+
+        </div>
+    </div>
+
+
+
+    <footer>
```
Comment (Benja-san) [inline contact.html:94]: why writing a footer when you have a footer.html file to include

File: formations.html [hunk 8]
```diff
@@ -0,0 +1,132 @@
...
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Formations</title>
+</head>
+
+<body>
+    <h1 class="titles center">Formations</h1>
+    <section class="categories">
+        <span>
```
Comment (Benja-san) [inline formations.html]: ul

File: formations.html [hunk 9]
```diff
@@ -0,0 +1,132 @@
...
+                <a href="">Comptabilité</a>
+                <a href="">Design</a>
```
Comment (Benja-san) [inline formations.html]: wrap a tags into li

File: formations.html [hunk 10]
```diff
@@ -0,0 +1,132 @@
...
+                <a href="">Finance</a>
+                <a href="">Informatique</a>
+                <a href="">Santé</a>
+                <a href="">Marketing </a>
+                </span>
+    </section>
+    <div class="formations">
```
Comment (Benja-san) [inline formations.html]: section

File: formations.html [hunk 11]
```diff
@@ -0,0 +1,132 @@
...
+        <h1 class="titles">Nos formations en </h1>
```
Comment (Benja-san) [inline formations.html]: h2 ?

File: formations.html [hunk 12]
```diff
@@ -0,0 +1,132 @@
...
+        <!--container of formation-->
+        <div class="container">
```
Comment (Benja-san) [inline formations.html]: use ul and li to list all courses

File: formulaire.html [hunk 13]
```diff
@@ -0,0 +1,106 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
```
Comment (Benja-san) [inline formulaire.html]: name the page title

File: formulaire.html [hunk 14]
```diff
@@ -0,0 +1,106 @@
...
+</head>
+
+<body>
+    <div class="formImage">
+        <img src="img\image-11.png" alt="">
+    </div>
+    <!--the first div of formulaire-->
+    <div class="formulaire">
+        <h2 class="titles">Python developper</h2>
```
Comment (Benja-san) [inline formulaire.html]: h1

File: formulaire.html [hunk 15]
```diff
@@ -0,0 +1,106 @@
...
+        <!--description of formation&-->
+        <p>
+            Cette formation a pour objectif de permettre de maîtriser la syntaxe essentielle de Python. Etre capable de créer et maintenir des applications complètes et complexes en Python
+        </p>
+        <!--finishing the description-->
+
+
+
+        <!--container of a propos -->
+        <div class="apropos">
```
Comment (Benja-san) [inline formulaire.html]: ul and li since it is a list of arguments

File: formulaire.html [hunk 16]
```diff
@@ -0,0 +1,106 @@
...
+                </span>
+            </div>
+            <!--fin de small container -->
+        </div>
+    </div>
+    <h2 class="titles">Postuler a cette formation</h2>
+    <form class="register" action="#" method="post">
+        <fieldset>
+            <legend>Nom et Prenom</legend>
+            <input type="text" name="name" id="">
```
Comment (Benja-san) [inline formulaire.html]: delete empty attributes

File: formulaire.html [hunk 17]
```diff
@@ -0,0 +1,106 @@
...
+        </fieldset>
+        <fieldset>
+            <legend>Message</legend>
+            <textarea name="message" id="" cols="30" rows="10">
+
+            </textarea>
+        </fieldset>
+        <input type="submit" value="S'inscrire">
+    </form>
+    <footer>
```
Comment (Benja-san) [inline formulaire.html:108]: include your footer.html

File: index.html [hunk 18]
```diff
@@ -0,0 +1,294 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/all.min.css" integrity="sha384-dfZ6qOWGZ3VFXG9g+Zl/54lJM2SXuL5bFP9Zl2zRGzFNM2a8L9Rtr4M+Kjq1rw2" crossorigin="anonymous">
+
+    <title>Oxygen School : formations en plusieurs discipline</title>
+</head>
+
+<body>
+
+    <nav>
```
Comment (Benja-san) [inline index.html:15]: use header and nav tags

File: index.html [hunk 19]
```diff
@@ -0,0 +1,294 @@
...
+                <h3>Comptabilité</h3>
+                <p>
+                    Formez-vous aux métiers de la comptabilité à travers nos différentes formations.
+                </p>
+            </div>
+        </div>
+    </nav>
+
+
+    <div class="description-choisir">
```
Comment (Benja-san) [inline index.html:83]: html semantic like sectionb, ul li.....

File: public/img/oxygenForMe.code-workspace [hunk 20]
```diff
@@ -0,0 +1,11 @@
+{
```
Comment (Benja-san) [inline public/img/oxygenForMe.code-workspace]: do you need this file ?

File: .hintrc [hunk 21]
```diff
@@ -0,0 +1,13 @@
+{
```
Comment (Benja-san) [inline .hintrc]: do you need this file ?

**Review by Benja-san — CHANGES_REQUESTED** [review]

File: formulaire.html (hunk 17 above)
Comment (yazidsef) [reply formulaire.html:108]: do you mean to code <?php include"footer.html" ?> !!

File: contact.html (hunk 3 above)
Comment (yazidsef) [reply contact.html]: yes i will but name i will use it with the  backend ... !!

**Review by Benja-san — APPROVED** [review]

**Pull request merged**